docs/content/management.md

```diff
@@ -267,6 +273,139 @@ you intended and that they are structured correctly. For example:
 opa run bundle.tar.gz
 ```
 
+### Signing
+
+To ensure the integrity of policies (ie. the policies are coming from a trusted source), policy bundles may be
+digitally signed so that industry-standard cryptographic primitives can verify their authenticity.
+
+OPA supports digital signatures for policy bundles. Specifically, a signed bundle is a normal OPA bundle that includes
+a file named `.signatures.json` that dictates which files should be included in the bundle, what their SHA hashes are,
+and of course is cryptographically secure.
+
+When OPA receives a new bundle, it checks that it has been properly signed using a (public) key that OPA has been
+configured with out-of-band.  Only if that verification succeeds does OPA activate the new bundle; otherwise, OPA
+continues using its existing bundle and reports an activation failure via the status API and error logging.
+
```

Comment:
Can we add a note here to warn folks that bundle signing is only supported in some modes? Something like this:

```
> ⚠️ Bundle signature verification is only performed by the `opa run` when the `-b`/`--bundle` flag is given or when Bundle downloading is enabled. Sub-commands primarily used in development and debug environments (such as `opa eval`, `opa test`, etc.) DO NOT verify bundle signatures at this point in time.
```

I just want to make sure we have something we can point to for the time being. We can work on introducing CLI support for verification in the future.

Reply:
Added the comment and squashed the commits.